common/rya.api/src/main/java/org/apache/rya/api/persist/index/RyaSecondaryIndexer.java

```diff
@@ -19,48 +19,46 @@
  * under the License.
  */
 
-
 import java.io.Closeable;
 import java.io.Flushable;
 import java.io.IOException;
 import java.util.Collection;
 import java.util.Set;
 
 import org.apache.hadoop.conf.Configurable;
-import org.openrdf.model.URI;
-
 import org.apache.rya.api.domain.RyaStatement;
 import org.apache.rya.api.domain.RyaURI;
+import org.eclipse.rdf4j.model.IRI;
 
 public interface RyaSecondaryIndexer extends Closeable, Flushable, Configurable {
```

Comment:
Switching all these interface methods to default access level is making the review rather large (which will still be public anyways).